vllm/sampling_params.py

```diff
@@ -261,4 +261,6 @@ def __repr__(self) -> str:
             f"prompt_logprobs={self.prompt_logprobs}, "
             f"skip_special_tokens={self.skip_special_tokens}, "
             "spaces_between_special_tokens="
-            f"{self.spaces_between_special_tokens})")
+            f"{self.spaces_between_special_tokens}, "
+            "logits_processors="
+            f"{self.logits_processors})")
```

Comment:
Could you remove this change from the PR? It would be good to handle in a separate PR since I'm not sure we necessarily want to log the string form of all of the logits processors (which could be arbitrary).